package.json

```diff
@@ -12,14 +12,15 @@
     "doctests",
     "test"
   ],
-  "main": "./lib/doctest",
+  "type": "module",
+  "main": "./lib/doctest.js",
   "bin": "./bin/doctest",
   "repository": {
     "type": "git",
     "url": "git://github.com/davidchambers/doctest.git"
   },
   "engines": {
-    "node": ">=10.2.0"
+    "node": ">=12.22.0"
```

Comment:
[Node v12.22.0][1] was the first version to support ES modules without requiring a command line flag.


[1]: https://nodejs.org/en/blog/release/v12.22.0/